base_tier_validation/models/tier_validation.py

```diff
@@ -732,7 +744,9 @@ def request_validation(self):
         return created_trs
 
     def _notify_restarted_review_body(self):
-        return _("The review has been reset by %s.") % (self.env.user.name)
+        return self.env._(
+            "The review has been reset by %(user_name)s.", user_name=self.env.user.name
+        )
```

Comment:
Same as previous comments. It's a shame to invalidate existing translatable terms just to use a kword argument on a term with only 1 arg